easybuild/tools/module_generator.py

```diff
@@ -622,7 +622,7 @@ def _generate_extensions_list(self):
         """
         Generate a list of all extensions in name/version format
         """
-        return self.app.make_extension_string(name_version_sep='/', ext_sep=',').split(',')
+        return self.app.make_extension_list(name_version_sep='/')
```

Comment:
Not sure this is the right approach, since it effectively bypasses `make_extension_string`, so easyblock that implement a custom `make_extension_string` won't work as expected anymore...

Reply:
1) I can't find a single easyblock (merged or in a PR) that touches `make_extension_string`
2) I have no clue how to else do this. The original code is weird as it goes from list->string->list.
3) the only use of `make_extension_string` is for generating the EBEXTLIST variables in the module. Totally different case so it would be good to split it.